### PR TITLE
Add background colors to corrections

### LINKED ISSUE #205: Background colors in corrections
As a user, I really miss the light background colors that visually separated the corrections in the old site: light red for wrong sentences and light green for correct ones. Now they all share the same white background which makes it more difficult to tell them apart, and also causes eye fatigue.
I can't find a picture of the old interface, perhaps @danielzeljko can provide one as reference?
image

### PR DESCRIPTION
**Pull request by keyxcode**
closes #205 

- I decided to make the text color for perfect sentences black (primary) because I thought it's easier to look at on green background.
- One thing I'm unsure about is the muted text for extra comments. I think it looks fine for short comments, but a bit hard to follow for longer ones. Let me know what you think @danielzeljko

### CODE REVIEW

File: langcorrect/templates/corrections/partials/correction.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div class="p-3 border-bottom">
-  <p class="{% if correction.type == 'perfect' %} text-success {% endif %} mb-0">
+<div class="p-3 {% if correction.type == 'perfect' %} bg-success bg-opacity-25 {% else %} bg-warning bg-opacity-25 {% endif %}">
```
Comment (danielzeljko) [inline langcorrect/templates/corrections/partials/correction.html:1]: I feel that the yellow background is quite overwhelming, especially when they're placed consecutively next to each other. It tends to draw too much attention away. I'm not sure what colors would work in this context, but after playing around for a little bit, I managed to come up with this:

image

If this version still feels harsh on the eyes, we could consider using a creamy or greyish color to compliment the green background without drawing too much attention away. I'm curious about which colors would achieve this balance, This would probably be better to discuss in a call so there's less back-and-forth communication on here.

**Review by danielzeljko — CHANGES_REQUESTED** [review]

**Comment by danielzeljko** [comment]
> One thing I'm unsure about is the muted text for extra comments. I think it looks fine for short comments, but a bit hard to follow for longer ones. Let me know what you think @danielzeljko

I felt the same way, so I attempted to address this in the screenshot above. What I did was remove the `text-secondary` and added `bg-secondary bg-opacity-10` instead.

**Comment by keyxcode** [comment]
Thanks for the feedback @danielzeljko
So after going through your suggestions, I've kinda synthesized them with my own ideas and come up with this version with the following changes:
- Removed the space around corrections and instead used a border for separation. I opted for the default border instead of a white border.
- Used `bg-light` for sentences that need correction. As for the extra comments, I decided to use the light yellow bg there instead of grey as you suggested. To me grey with green and white look a bit too cold and metalli while yellow adds a nice warm accentuation -- and it's also reminiscent of the original design. 

Pls lmk what you think!

image

**Comment by danielzeljko** [comment]
@keyxcode 

Your changes look great! 💯 I don't know why I seem to have an affinity to post-apocalyptic LC 🤣

File: langcorrect/templates/corrections/partials/user_correction_card.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 <div class="card">
   {% include "posts/partials/card_header.html" %}
-  <div class="d-grid gap-1 p-1">
+  <div class="d-grids">
```
Comment (danielzeljko) [inline langcorrect/templates/corrections/partials/user_correction_card.html:7]: There's a typo here -- it should be `d-grid`, but I think we can most likely leave this out completely

**Review by danielzeljko — CHANGES_REQUESTED** [review]

**Review by danielzeljko — APPROVED** [review]
